src/pages/nutrition/index.jsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+// import NutritionTable from "./table";
+import ProductNutriments from "./productCard";
+import { Box } from "@mui/material";
+
+// import { basicNutriments } from "./nutritionFields";
+
+export default function Nutrition() {
+  /*const [nutriments, setNutriments] = React.useState(basicNutriments);
+
+  function onchangeHandler(e) {
```

Comment:
nit: please remove commented code

Reply:
@Shmigolk Please mark a comment as resolved only after pushing the fix, such that we can follow what as been said.